src/main/java/de/retest/recheck/ui/diff/Warning.java

```diff
@@ -0,0 +1,33 @@
+package de.retest.recheck.ui.diff;
+
+import javax.xml.bind.annotation.XmlAttribute;
+
+public class Warning {
```

Comment:
You can also use `@Data` to avoid all this boilerplate code, we have Lombok already activated in recheck.